src/subject_level/getFFXdir.m

```diff
@@ -25,11 +25,15 @@
     glmDirName = [glmDirName, '_desc-', convertToValidCamelCase(model.Name)];
   end
 
-  ffxDir = fullfile(opt.derivativesDir, ...
+  ffxDir = fullfile(opt.dir.stats, ...
                     ['sub-', subLabel], ...
                     'stats', ...
                     glmDirName);
 
+  if opt.glm.roibased.do
```

Comment:
neat!

Reply:
Yeah that's pretty much it: one of the few thing this flag does is append a suffix to the GLM folder.

This way in this case you can use the same model json for both whole brain and roi based analysis